_python/test_helpers.py

```diff
@@ -1,7 +1,27 @@
+from django.test.testcases import SimpleTestCase
 from rest_framework.response import Response
+from urllib.parse import urljoin, urlsplit
 
 
-def check_response(response, status_code=200, content_type=None, content_includes=None, content_excludes=None):
+def assertURLEqual(response, expected_url):
```

Comment:
Nit -- does this want to be `assert_url_equal`?

Reply:
yep.